sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -446,4 +452,84 @@ export class InstitutionController extends BaseController {
       },
     }));
   }
+
+  /**
+   * Get the Institution details for the ministry institution detail page
+   * @param institutionId
+   * @returns AESTInstitutionDetailDto
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Groups(UserGroups.AESTUser)
+  @Get("/:institutionId/detail")
+  async getAESTInstitutionDetailById(
+    @Param("institutionId") institutionId: number,
+  ): Promise<AESTInstitutionDetailDto> {
+    if (!institutionId) {
```

Comment:
I believe that if the id is not provided the route is not going to be satisfied hence this controller will not even be called.
Can you please check? I believe that this validation is not necessary.